bitnami/concourse/templates/_helpers.tpl

```diff
@@ -0,0 +1,153 @@
+{{/*
+Return the proper web image name
+*/}}
+{{- define "concourse.image" -}}
+{{ include "common.images.image" (dict "imageRoot" .Values.concourse.image "global" .Values.global) }}
+{{- end -}}
+
+{{/*
+Return the proper image name (for the init container volume-permissions image)
+*/}}
+{{- define "concourse.volumePermissions.image" -}}
+{{- include "common.images.image" ( dict "imageRoot" .Values.volumePermissions.image "global" .Values.global ) -}}
+{{- end -}}
+
+{{/*
+Return the proper Docker Image Registry Secret Names
+*/}}
+{{- define "concourse.imagePullSecrets" -}}
+{{- include "common.images.pullSecrets" (dict "images" (list .Values.concourse.image .Values.volumePermissions.image) "global" .Values.global) -}}
+{{- end -}}
+
+
+{{/*
+Compile all warnings into a single message.
+*/}}
+{{- define "concourse.validateValues" -}}
+{{- $messages := list -}}
+{{- $messages := append $messages (include "concourse.validateValues.enabeled" .) -}}
+{{- $messages := without $messages "" -}}
+{{- $message := join "\n" $messages -}}
+
+{{- if $message -}}
+{{-   printf "\nVALUES VALIDATION:\n%s" $message -}}
+{{- end -}}
+{{- end -}}
+
+{{/* Check if web or worker are enable */}}
+{{- define "concourse.validateValues.enabeled" }}
+{{ if not (or .Values.web.enabled .Values.worker.enabled) }}
+concourse: enabeled
+  Must set either web.enabled or worker.enabled to create a concourse deployment
+{{ end }}
+{{- end -}}
+
+{{/*
+Return the appropriate apiVersion for deployment.
+Sometimes GitVersion will contain a `v` so we need
+to strip that out.
+*/}}
+{{- define "concourse.deployment.apiVersion" -}}
+{{- $version := include "concourse.kubeVersion" . -}}
+{{- if semverCompare "<1.9-0" $version -}}
+{{- print "extensions/v1beta1" -}}
+{{- else -}}
+{{- print "apps/v1" -}}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Create a default fully qualified web node(s) name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+*/}}
+{{- define "concourse.web.fullname" -}}
```

Comment:
I would expect something like this (example from cert-manager)

```

{{/*
Create a default fully qualified app name.
We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
If release name contains chart name it will be used as a full name.
*/}}
{{- define "certmanager.controller.fullname" -}}
{{- printf "%s-controller" (include "common.names.fullname" .) | trunc 63 | trimSuffix "-" -}}
{{- end -}}
```